External/Plugins/ASCompletion/PluginMain.cs

```diff
@@ -1209,10 +1209,9 @@ private void OnMouseHover(ScintillaControl sci, int position)
 
             // get word at mouse position
             int style = sci.BaseStyleAt(position);
-            if (!ASComplete.IsTextStyle(style))
-                return;
-            position = sci.WordEndPosition(position, true);
-            ASResult result = ASComplete.GetExpressionType(sci, position);
+            if (!ASComplete.IsTextStyle(style)) return;
+            position = ASComplete.ExpressionEndPosition(sci, position);
```

Comment:
From my understanding, when hovering a function call, this will provide the type of the return value and not the function member itself. That would be unexpected.

Reply:
![2018-01-08_17-43-08](https://user-images.githubusercontent.com/1700940/34675674-6cf2e9e2-f49b-11e7-96a8-48c8cacbdbf2.png)